### ISSUE TITLE
Rich paste support

### ISSUE THREAD

**Issue by mHatsune** [notable/notable#1219]
#### Current behavior

When copy some texts from oneone to notable then it paste as `![](@attachment/Clipboard_2020-07-05-02-26-47.png)`

**Comment by fabiospampinato** [comment]
It looks like you had an image in your clipboard? 🤔 Do you get the same issue when copying text from a browser for example?

**Comment by mHatsune** [comment]
Or enhance to display paste options

- Keep source formatting
- Merge formatting
- Keep text only
- Picture

**Comment by mHatsune** [comment]
![image](https://user-images.githubusercontent.com/63988538/86519754-f7be0580-be78-11ea-9f85-fdaf04e025d0.png)


> It looks like you had an image in your clipboard? 🤔 Do you get the same issue when copying text from a browser for example?

No

**Comment by fabiospampinato** [comment]
I don't think it's possible for the app to implement those pasting options, it's either pasting text or pasting an image.

Maybe what you would like want to have is the ability to import note directly from OneNote: https://github.com/notable/dumper/issues/13

**Comment by mHatsune** [comment]
@fabiospampinato Paste options are built-in feature on one note
Let's check bit, no such issues when copy from one note to notepad, wordpad, libreoffice

**Comment by mHatsune** [comment]
Furthermore, libreoffice is open source product
It also paste from one note as a image but it has paste special feature to paste as unformatted text

**Comment by fabiospampinato** [comment]
Yeah but we have some limitations that those other apps don't have, I don't think we can implement this 🤔 

I'll take a closer look at this though, maybe I'm wrong.

For future reference: https://www.electronjs.org/docs/api/clipboard#clipboardreadhtmltype

**Comment by mHatsune** [comment]
> Yeah but we have some limitations that those other apps don't have, I don't think we can implement this 🤔
> 
> I'll take a closer look at this though, maybe I'm wrong.
> 
> For future reference: https://www.electronjs.org/docs/api/clipboard#clipboardreadhtmltype

Wow. this already solved on 1.9.0-alpha.14 and ago i used 1.8.4 but you need keep this opened to implement above mentioned paste options
Alpha release has another problem, it doesn't paste clipboard images to notable when press ctrl+v however ago it only processed as markdown attachment but this feature more important https://github.com/notable/notable/issues/1221 > #284 than it so i dont consider as a bug for now.
